homework7/controller.py

```diff
@@ -0,0 +1,101 @@
+import model
+import view
+
+files = model.f
+
+def run():
+    running = True
+    while running:
+        task = view.operation()
+        if task == 1:
+            create = model.create_directory()
+            if create not in files:
+                files.append(create)
+                answer1 = view.create_success()
+                if answer1 == 1:
+                    model.add_data(create)
+                if answer1 == 3:
+                    exit()
```

Comment:
Я говорил, что если вы хотите завершить программу подобным образом, то лучше делать так:
```suggestion
                    sys.exit()
```
Хотя здесь лучше всего подходит использование банального `break`.